js/views/conversation_view.js

```diff
@@ -366,15 +366,17 @@
 
         removeScrollDownButton: function() {
             if (this.scrollDownButton) {
-                this.scrollDownButton.remove();
+                var button = this.scrollDownButton;
                 this.scrollDownButton = null;
+                button.remove();
```

Comment:
Not exactly sure. Received a report that the last seen indicator didn't go away on send. This will ensure that if the `.remove()` crashes, we'll still set `this.lastSeenIndicator = null`.